src/Cms/System.php

```diff
@@ -286,21 +286,24 @@ public function license(): string|bool
 			$license['date'],
 			$license['email'],
 			$license['domain'],
-			$license['signature']
+			$license['signature'],
+			$license['type'],
```

Comment:
I think it would be good to also include the revenue limit in the license data (and JSON for the signature) so we can display it in the license dialog along with an explanation about the license type.

Reply:
Isn't this implied via the type?

Reply:
Then we would need to hardcode the limit in the core though.

Reply:
I think both ways have their problems. If we hardcode it in the license file, we cannot retroactively change it for existing licenses anymore without having wrong info in the dialog. Maybe it makes sense to not show this here, but in the hub?

Reply:
Since we don't offer a way (neither require) to switch during the 3 years the license if the conditions change, I think it actually doesn't make too much sense to show them during this time, does it? But only when buying and upgrading.

Reply:
We can never retroactively change the limit for existing licenses. Our customers have a contract with us at the time of their purchase.

The reason why I think it should be in the dialog: So that users know if their license is at all the right one. If a very large business has a Basic license, everyone with access to the system view will be able to see that it's not right, creating pressure to buy the right license in the first place.